### DISCUSSION TITLE
¿Quieres?

### DISCUSSION THREAD

**Discussion by pauket** [neugomonov/restik#4] [Q&A] [answered]
😳✍

**Comment by neugomonov** [comment, marked as answer]
No, mejor no. Deja de fumar. Es malo para ti. Tus amigos y tu familia, te quieren mucho. No desperdicies tu vida fumando un trozo de hierba por que aún eres jóven y tienes toda una vida por delante. Espero que replantees tus decisiones. Vamos, coge mi mano y andemos juntos hacia este atardecer, por qué yo solo quiero tu amistad.
